.github/actions/run-acceptance-test/action.yml

```diff
@@ -99,13 +102,17 @@ runs:
         GITLAB_SUBGROUP: weave-gitops-sub
         GITLAB_TOKEN: ${{ inputs.gitlab-token }}
         GITLAB_KEY: ${{ inputs.gitlab-key }}
+        GIT_PROVIDER: ${{ inputs.git-provider }}
         ARTIFACTS_BASE_DIR: ${{ inputs.artifacts-base-dir }}
     run: |
       export PATH=${PATH}:`go env GOPATH`/bin
       export WEGO_BIN_PATH=$(pwd)/bin/gitops
       export CLUSTER_PROVIDER=kubectl
       go get github.com/onsi/ginkgo/ginkgo
       go get github.com/onsi/gomega/...
+      if [ "${{ github.job }}" == "smoke-tests-long" ]; then
+        export GIT_PROVIDER=${{ matrix.GIT_PROVIDER }}
```

Comment:
is it possible to use `inputs.git-provider` here to avoid having GIT_PROVIDER here https://github.com/weaveworks/weave-gitops/pull/1245/files#diff-faff1af3d8ff408964a57b2e475f69a6b7c7b71c9978cccc8f471798caac2c88R146

Reply:
If this is set here maybe we don't need this https://github.com/weaveworks/weave-gitops/pull/1245/files#diff-8552c4eceb0d8c09cef35ae019c005254bfe2042358e5acaac36dd9846c739e4R105 ?

Reply:
I tried that previously - it didn't work